src/transformers/models/dpt/modeling_dpt.py

```diff
@@ -0,0 +1,1130 @@
+                x = x.reshape(feature_shape)
+            else:
+                pass
+            x = self.projects[i](x)
+            x = self.resize_layers[i](x)
+            out.append(x)
+
+        return out
+
+
+class DPTPreActResidualConvUnit(nn.Module):
+    """
+    ResidualConvUnit, pre-activate residual unit.
+
+    Args:
+        in_channels (int): number of channels in the input feature map.
+        act_cfg (dict): dictionary to construct and config activation layer.
+        norm_cfg (dict): dictionary to construct and config norm layer.
+        stride (int): stride of the first block. Default: 1
+        dilation (int): dilation rate for convs layers. Default: 1.
+        init_cfg (dict, optional): Initialization config dict. Default: None.
+    """
+
+    def __init__(self, config):
+        super().__init__()
+
+        self.use_batch_norm = config.use_batch_norm
+        self.act1 = nn.ReLU()
+        self.conv1 = nn.Conv2d(
+            config.channels,
+            config.channels,
+            kernel_size=3,
+            stride=1,
+            padding=1,
+            bias=not self.use_batch_norm,
+        )
+
+        self.act2 = nn.ReLU()
+        self.conv2 = nn.Conv2d(
+            config.channels,
+            config.channels,
+            kernel_size=3,
+            stride=1,
+            padding=1,
+            bias=not self.use_batch_norm,
+        )
+
+        if self.use_batch_norm:
+            self.batch_norm1 = nn.BatchNorm2d(config.channels)
+            self.batch_norm2 = nn.BatchNorm2d(config.channels)
+
+    def forward(self, inputs):
+        inputs_ = inputs.clone()
+        x = self.act1(inputs)
+
+        x = self.conv1(x)
+
+        if self.use_batch_norm:
+            x = self.batch_norm1(x)
+
+        x = self.act2(x)
+        x = self.conv2(x)
+
+        if self.use_batch_norm:
+            x = self.batch_norm2(x)
+
+        return x + inputs_
+
+
+class DPTFeatureFusionBlock(nn.Module):
+    """FeatureFusionBlock, merges feature maps from different stages.
+
+    Args:
+        config (dict): config dict.
+        expand (bool): Whether to expand the channels in the post process block.
+            Default: False.
+        align_corners (bool): align_corner setting for bilinear upsample.
+            Default: True.
+    """
+
+    def __init__(self, config, expand=False, align_corners=True):
+        super().__init__()
+
+        self.expand = expand
+        self.align_corners = align_corners
+
+        self.out_channels = config.channels
+        if self.expand:
+            self.out_channels = config.channels // 2
+
+        out_channels = config.channels // 2 if self.expand else config.channels
+        self.project = nn.Conv2d(config.channels, out_channels, kernel_size=1, bias=True)
+
+        self.res_conv_unit1 = DPTPreActResidualConvUnit(config)
+
+        self.res_conv_unit2 = DPTPreActResidualConvUnit(config)
+
+    def forward(self, *inputs):
```

Comment:
Why accept and ignore more than one input? This is going to lead to problems that are hard to debug. This should only accept one input.